app/javascript/app/components/modal.js

```diff
@@ -1,48 +1,47 @@
 import 'classlist.js';
+import { createFocusTrap } from 'focus-trap';
 import Events from '../utils/events';
 
 const STATES = {
   HIDE: 'hide',
   SHOW: 'show',
 };
 
-function modal(focusTrap) {
-  return class extends Events {
-    constructor(options) {
-      super();
+class Modal extends Events {
+  constructor(options) {
+    super();
 
-      this.el = document.querySelector(options.el);
```

Comment:
This is all indentation changes. Suggest to review with "Hide whitespace changes":

![image](https://user-images.githubusercontent.com/1779930/93373895-fb301e80-f823-11ea-9093-e0089333634a.png)